examples/api/csharp/rcs/Rcs-Api-Console-Tester/RcsConsoleApp/RcsCodeSamples.cs

```diff
@@ -127,7 +127,7 @@ public static async Task ChangeSectionReiforcedSection(RcsApiClient client)
 		{
 			#region changereinforcedcrosssection
 			//Get the list of avaliable reinforced cross-sections in the project
-			List<ReinforcedCrossSectionModel> reinforcedCrossSections = await client.GetProjectReinforcedCrossSectionsAsync(CancellationToken.None);
+			List<RcsReinfCssModel> reinforcedCrossSections = await client.GetProjectReinforcedCrossSectionsAsync(CancellationToken.None);
```

Comment:
Personally I do not aggree

RcsSectionResultDetailed makes sense

but
RcsReinfCss not. The correct name which we use in our concrete terminology is ReinforcesCrossSection and it is a part of our structural model

https://dev.azure.com/ideastatica/_git/IdeaStatiCa?path=/Services/Concrete/ServicePoint/ReinforcedSection/ReinfSection.cs

where the class is called ReinfSection - Prefix Rcs is not needed (according to my understanding) - Let's discuss it with Natahan. If he thinks RcsReinfCssModel is a nice name we will merge it